### PR TITLE
Boevski/rework reorg

### PR DESCRIPTION
**Pull request by boevski**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,52 +1,100 @@
...
+* [Overview](#overview)
+* [Screenshots](#screenshots)
+* [Requirements](#requirements)
+* [Configuration](#configuration)
+* [Running the Sample](#running-the-sample)
 
-Here is a list of the features that are showcased in the Friends sample app:
+## Overview
+
+This repository contains the Telerik Friends app for PhoneGap/Cordova. It is a sample mobile app demonstrating how to integrate a large gamut of Telerik Platform services into a native mobile application using the [NativeScript](http://github.com/nativescript/nativescript) framework.
```
Comment (ikoevska) [inline README.md]: "for PhoneGap/Cordova" should probably be NativeScript.

File: README.md [hunk 2]
```diff
@@ -1,52 +1,100 @@
...
+
+The Telerik Friends sample app showcases these features and SDKs:
 
 - Cloud data access (Telerik Backend Services)
```
Comment (ikoevska) [inline README.md]: I would make all of these links to the corresponding docs.

File: README.md [hunk 3]
```diff
@@ -1,52 +1,100 @@
...
+---|---|---
+![Login Screen](https://raw.githubusercontent.com/Icenium/nativescript-sample-Friends/master/screenshots/android-login-screen.png)|![Activities stream view](https://raw.githubusercontent.com/Icenium/nativescript-sample-Friends/master/screenshots/android-activities-stream.png)|![Activity details view](https://raw.githubusercontent.com/Icenium/nativescript-sample-Friends/master/screenshots/android-activity-details.png)
+
+## Requirements
 
-## Requirements  
+Before you begin, you need to ensure that you have the following:
 
-The following is a list of requirements for the sample app:
+- **An active Telerik Platform account**
```
Comment (ikoevska) [inline README.md]: This is not really true. If you don't have an account, as soon as you click Try in AppBuilder, you get prompted to create an account after which the sample gets cloned.

File: README.md [hunk 4]
```diff
@@ -1,52 +1,100 @@
...
+Ensure that you can log in to a Telerik Platform account. This can be a free trial account. Depending on your license you may not be able to use all app features. For more information on what is included in the different editions, check out the pricing page. All features included in the sample app work during the free trial period.
 
-- **Active Telerik Platform account**  
-To use this sample app you must have an active Telerik Platform account. Depending on your license you may not be able to use all features of the app. For more information on what is included in the different editions, please check out the pricing page for the respective product. All features included in the sample app will work in the free trial period. Go to [http://platform.telerik.com](http://platform.telerik.com) for more info.
+- **Telerik AppBuilder** The sample app requires Telerik AppBuilder to run. This can be the in-browser client, the desktop client or the extension for Visual Studio.
```
Comment (ikoevska) [inline README.md]: Or the command-line interface.

File: README.md [hunk 5]
```diff
@@ -1,52 +1,100 @@
...
+By default the Friends Sample is configured to use HTTP. If you want to switch to HTTPS, update the `global.BS_SCHEME` setting in  `Friends/app/app.js` to `https`.
+
+## Running the Sample
+
+Once the app is configured, you can run it either on a real device or a native emulator.
+
+To run it, follow the steps in the product's documentation: [Running Apps on Devices](http://docs.telerik.com/platform/appbuilder/testing-your-app/running-on-devices/working-with-devices).
+
+> Ensure that the emulator or the device that you are using has Internet connectivity when running the sample.
+
```
Comment (ikoevska) [inline README.md]: What do you need the empty ~10 lines for?

File: README.md (hunk 1 above)
Comment (boevski) [inline README.md]: :+1: Fixed.

File: README.md (hunk 2 above)
Comment (boevski) [inline README.md]: :+1: Fixed.

File: README.md (hunk 3 above)
Comment (boevski) [inline README.md]: Thanks, I didn't know that. Nevertheless, it wouldn't hurt creating an account beforehand. We can't document all our features in a single paragraph :-)

File: README.md (hunk 4 above)
Comment (boevski) [inline README.md]: :+1: Added.

File: README.md (hunk 5 above)
Comment (boevski) [inline README.md]: It's not intentional :-)

**Pull request merged**